splitter/use-100-percent-viewport/readme.md

```diff
@@ -0,0 +1,19 @@
+# How to make Splitter take 100% height of the viewport
+
+When creating layouts with the splitter, it is a common requirement that they take up 100% of the available height.
+
+To accomplish this, simply set the Splitter's `Height` parameter to `100%`.
+
+It is important to note that this relies on the standard CSS behavior of setting height in percentage - all parent elements must have a `height` set, so that the content size can be calculated based off the viewport.
```

Comment:
```suggestion
It is important to note that this relies on the standard CSS behavior of setting height in percent - all parent elements must have a `height` set, so that the content size can be calculated based off the viewport.
```